gears/src/application/node.rs

```diff
@@ -65,25 +54,27 @@ impl<'a, Core: Node, AI: ApplicationInfo> NodeApplication<'a, Core, AI> {
     ) -> Self {
         Self {
             core,
-            router: Core::router(),
             abci_handler_builder,
             params_store_key,
             params_subspace_key,
         }
     }
 
     /// Runs the command passed on the command line.
-    pub fn execute(self, command: AppCommands<Core::AuxCommands>) -> anyhow::Result<()> {
+    pub fn execute<AI: ApplicationInfo>(
```

Comment:
It's still used by the `info` method. The `ApplicationInfo` was a generic argument for the `RestState`. Now that it isn't we get an error that the generic is not used, so I pass it in as part of the execute command rather than using PhantomData.

Reply:
Okay